search_server.py

```diff
@@ -201,7 +207,7 @@ def search(self, q: str, n: int) -> Generator[str, None, None]:
 
 class GoogleSearchServer(SearchABC):
     def search(self, q: str, n: int) -> Generator[str, None, None]:
-        return googlesearch.search(q, num=n, stop=None, pause=_DELAY_SEARCH)
+        return googlesearch.search(q, num=n, stop=None, pause=_DELAY_SEARCH, tbs="qdr:m")
```

Comment:
can you remove `tbs="qdr:m"` please, we don't want pages from just this month to show up

Reply:
https://github.com/MarioVilas/googlesearch/blob/master/googlesearch/__init__.py#L221